### ISSUE TITLE
remove handleFocusNextHotKey and handleFocusPreviousHotKey

### ISSUE THREAD

**Issue by macrozone** [react-page/react-page#766]
while updating slate i saw that plugins can implement handleFocusNextHotKey and handleFocusPreviousHotKey. It should enable to navigate between cells of the editor with keyboard (i guess?)

 but most plugins don't  implement it and its not so trivial to reimplement it with the new slate at the moment, so I would propose to remove it, because it is not worth the effort. It would also help reducing the api surface.

thoughts?

**Comment by macrozone** [comment]
ok, while refactoring, i found out that this functionality was indeed broken ^^ I fixed it, but i thought that its probably a bit annoying, if every plugin has to decide whether to catch it. 

The hotkeys where: del, backspace and the arrow keys (and the usual undo / redo hotkeys).
While del and backspace are debateable, the arrow keys are used so often and ambiguously, that i think its better to prefix them with a modifier. I currently went with alt+, as this combination is not used so often.  I also prefixed del and backspace with alt. alt+backspace is usually delete word (at least on my mac), but i am not sure if its used often (i acutally did not know until i tried it).

I would integrate that into the slate branch, as its affects the hotkey handling of slate anyway.
